x-pack/filebeat/processors/aws_vpcflow/parse_aws_vpc_flow_log.go

```diff
@@ -36,10 +36,7 @@ var (
 
 type processor struct {
 	config
-	fields             []vpcFlowField
-	log                *logp.Logger
-	originalFieldCount int
-	expectedIPCount    int
+	formats []*formatProcessor
```

Comment:
Does this need to be a slice of pointer?

Reply:
Thank you! I changed it to store `[]formatProcessor`